source/pattern-library/data-visualization/heat-map/index.md

```diff
@@ -1,60 +1,8 @@
 ---
-title: Heat Map
-author: dlabrecq
-layout: page-tabs
+author: lhinson
+layout: page-pattern
+overview: patterns/data-visualization/heat-map/overview.md
+design: patterns/data-visualization/heat-map/design.md
+code_html: false
+code_angular: /components/angular-patternfly/dist/docs/partials/api/patternfly.charts.directive.pfHeatMap.html
```

Comment:
I see. I understand that Dave wrote the ng-include directive to include the angular partials. You might want to have a chat with him?